bin/run.py

```diff
@@ -328,9 +328,9 @@ def run(self, in_path, out_path, crop=True, args=[], cwd=None, default_timeout=F
     # Run this submission on all testcases for the current problem.
```

Comment:
update the comment to match that `testcases` is passed in now, or maybe rename to just `run_testcases`, since it's only called from `run_some`.